src/session9/challenges/college_management_system/Student.java

```diff
@@ -0,0 +1,81 @@
+package session9.challenges.college_management_system;
+
+public class Student {
+
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private int age;
+    private String dateOfBirth;
+    private String CNP;
+    private String address;
```

Comment:
Student and Professor classes have the same properties, so we end up having duplicated code. We can do something about this to optimize it?